int8 START=0
int8 STOP=1
int8 RESET=2

int8 type
string traj_name
bool is_trajectory
bool is_forward
bool is_setup
